src/core/service.c

```diff
@@ -1093,6 +1105,21 @@ static void service_dump(Unit *u, FILE *f, const char *prefix) {
                         fprintf(f, "%sOpen File: %s\n", prefix, ofs);
                 }
 
+        if (s->extra_fds)
```

Comment:
This check can be dropped, `LIST_FOREACH()`  will be a noop if `extra_fds` is `NULL`